src/main/resources/db/migration/common/V1_403__add_user_for_automated_task.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO user_account VALUES (-5, NULL, 'AudioLinkingAutomatedTask', NULL, 'AudioLinkingAutomatedTask', '2024-01-01 00:00:00+00', '2024-01-01 00:00:00+00', NULL, 0, 0, NULL, true, true, 'AudioLinkingAutomatedTask');
```

Comment:
This can be removed after Ben's change goes in https://github.com/hmcts/darts-api/pull/2226

Reply:
Removed